sql/structured.go

```diff
@@ -242,6 +242,16 @@ func (desc *TableDescriptor) FindColumnByID(id ColumnID) (*ColumnDescriptor, err
 	return nil, fmt.Errorf("column-id \"%d\" does not exist", id)
 }
 
+// GetIndexByName finds the index with specified name.
+func (desc *TableDescriptor) GetIndexByName(name string) *IndexDescriptor {
```

Comment:
This method should follow the example set by `FindColumnBy{Name,ID}` in terms of its naming and its return (i.e. return an error when not found).